### DISCUSSION TITLE
Multiple artist support question

### DISCUSSION THREAD

**Discussion by Thornado007** [navidrome/navidrome#2614] [Q&A]
Hi I just read through the "Support multiple artists, with custom separator #238" "Multiple artist support #2524" thread and this seemed to answer all my questions surrounding how navidrome handles multiple artists. However when I go to the web client of navidrome the album "watch the throne" by Jay-z and Kanye west is still shown as one artist "Jay-z & Kanye west" :
![image](https://github.com/navidrome/navidrome/assets/80928962/58c43551-437f-4f9f-bc89-df34a083ef17)

Ideally I would like that they are shown as separate artists like it seems to be the case here: 
![image](https://github.com/navidrome/navidrome/assets/80928962/291bef39-04dc-45b9-a17c-ccfd4bd7f263)

I tag my music using Musicbrainz Picard and the tags for a song of the albums look like this:
![image](https://github.com/navidrome/navidrome/assets/80928962/ace53f10-766e-4054-8ccb-be2df00a1500)

I installed Navidrome last week using docker on my synology NAS. I use Feishin on my PC and Symfonium on my android device. Other than the artist separation it works wonderfully and am very gratefull for it.

Is there something I can do to make the Artists separate? maybe use another version of navidrome? I'm quite a noob at this stuff and already tried changing a lot of the tags and adding taglib as an environment variable but nothing seems to work. Thanks in advance!

**Comment by viazure** [comment]
I found this [WIP: Big Refactor (new scanner and DB schema changes)](https://github.com/navidrome/navidrome/pull/2709), and it looks like this work is still in progress, with multiple PRs not really finished merging yet. I'm sure this should be coming soon, so keep watching along.
